GitExtensions/Program.cs

```diff
@@ -44,16 +43,16 @@ private static void Main()
 
             AppSettings.SetDocumentationBaseUrl(ThisAssembly.Git.Branch);
 
-            ThemeModule.Load();
-            Application.ApplicationExit += (s, e) => ThemeModule.Unload();
+            ////ThemeModule.Load();
+            ////Application.ApplicationExit += (s, e) => ThemeModule.Unload();
```

Comment:
💭 I worry that it will be particularly difficult to track down the commented theming code. I almost wish it was wrapped in `#if SUPPORT_THEMES` instead of just commented.